files/en-us/web/javascript/reference/global_objects/set/index.html

```diff
@@ -209,11 +209,11 @@ <h3 id="Implementing_basic_set_operations">Implementing basic set operations</h3
 const setB = new Set([2, 3])
 const setC = new Set([3, 4, 5, 6])
 
-isSuperset(setA, setB)          // =&gt; true
-union(setA, setC)               // =&gt; Set [1, 2, 3, 4, 5, 6]
-intersection(setA, setC)        // =&gt; Set [3, 4]
-symmetricDifference(setA, setC) // =&gt; Set [1, 2, 5, 6]
-difference(setA, setC)          // =&gt; Set [1, 2]
+isSuperset(setA, setB)          // returns true
+union(setA, setC)               // returns {1, 2, 3, 4, 5, 6}
+intersection(setA, setC)        // returns {3, 4}
+symmetricDifference(setA, setC) // returns {1, 2, 5, 6}
+difference(setA, setC)          // returns {1, 2}
```

Comment:
Worth keeping `Set` in:

```suggestion
isSuperset(setA, setB)          // returns true
union(setA, setC)               // returns Set {1, 2, 3, 4, 5, 6}
intersection(setA, setC)        // returns Set {3, 4}
symmetricDifference(setA, setC) // returns Set {1, 2, 5, 6}
difference(setA, setC)          // returns Set {1, 2}
```

Reply:
Nice